app/src/main/java/com/garethevans/church/opensongtablet/PopUpPageButtonsFragment.java

```diff
@@ -150,7 +151,8 @@ public void onClick(View view) {
         custom3Visible_Switch.setText(c3);
         custom4Visible_Switch.setText(c4);
 
-        enableordisablegrouping();
+        // IV - collapsing and expanding groups means a block is no longer needed
+        //enableordisablegrouping();
```

Comment:
This was to stop grouping of information and custom buttons from being active if all buttons are grouped in one setting as changing them would have no effect on what is seen if all buttons are already grouped

Further review required

Reply:
This is consistent with the 'new' ability to expand/collapse groups.